docs/labs/ja_hardcoded.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://best.openssf.org/assets/css/style.css">
+<link rel="stylesheet" href="checker.css">
+<script src="checker.js"></script>
+<script src="hardcoded.js"></script>
+<link rel="license" href="https://creativecommons.org/licenses/by/4.0/">
+
+<!-- See create_labs.md for how to create your own lab! -->
+</head>
+<body>
+<!-- For GitHub Pages formatting: -->
+<div class="container-lg px-3 my-5 markdown-body">
+<h1>ラボ演習 hardcoded</h1>
+<p>
+これはセキュアなソフトウェア開発に関するラボ演習です。
+ラボの詳細については、<a href="ja_introduction.html" target="_blank">概要</a>をご覧ください。
+
+<p>
+<h2>タスク</h2>
+<p>
+<b>サンプルコードからハードコードされたクレデンシャル情報を取り除いてください。</b>
```

Comment:
How about using "認証情報" or "識別情報" (this might sound a little bit odd, though) instead of "クレデンシャル情報" for the translation of "credentials", again unless there's any intention?

Reply:
Agree.  "credential" = "認証情報".
```suggestion
<b>サンプルコードからハードコードされた認証情報を取り除いてください。</b>
```
Fixed multiple similar cases as below.

- `この演習では、ハードコードされた認証情報（この場合はパスワード）をコードから取り除きます。`
- `以下の Java コードを変更して、ハードコードされた認証情報を取り除いてください。`
- `パスワードなどの秘密とすべき認証情報がハードコードされていると、不必要に広く公開されてしまい、また変更が困難になります。`
- `<i>なにより認証情報をハードコードするよりも良い方法である</i> ことから、環境変数を用います。`